src/govuk/components/character-count/character-count.mjs

```diff
@@ -57,6 +72,12 @@ function CharacterCount ($module, config) {
     datasetConfig
   )
 
+  console.log(this.$module)
+  this.i18n = new I18n(extractConfigByNamespace(this.config, 'i18n'), {
+    // Read the fallback if necessary rather than have it set in the defaults
+    locale: this.config.i18nLocale || closestAttributeValue($module, 'lang')
```

Comment:
Picking up from the discussion in https://github.com/alphagov/govuk-frontend/pull/2887#discussion_r983423951, are we sure we need the `i18nLocale` config option? I think it's OK to always rely on the closest `lang` attribute – not 100% sure what the need is to be able to define it in JS.

Reply:
I'll remove the `i18nLocale` config option then. It was more there for parity than anything.